lib/readline.js

```diff
@@ -90,6 +90,12 @@ function Interface(input, output, completer, terminal) {
   this._previousKey = null;
   this.escapeCodeTimeout = ESCAPE_CODE_TIMEOUT;
 
+  // stuff used for reverse search
+  this.reverseSearchPrompt = "(reverse-i-search)`' ";
```

Comment:
This seems to be a constant. Please just use `const kReverseSearchPromt = '...'` in the outer scope.